### ISSUE TITLE
Content Blocks - Umbraco 10.2.0 introduces a breaking change

### ISSUE THREAD

**Issue by leekelleher** [leekelleher/umbraco-contentment#250]
### Which Contentment version are you using?

4.1.3

### Which Umbraco version are you using? For example: 8.14.1 - don't just write v8

10.2.0

### Bug summary

Upgrading to Umbraco v10.2.0, the Content Blocks overlay editor throws an error and does not display any of the property editors.

In the browser console, the following error is displayed...

```
TypeError: Cannot read properties of undefined (reading 'variants')
    at $scope.propertyEditorDisabled (umbraco.directives.min.js?d=637983178250000000:1:112602)
```

Expanding the code in `umbraco.directives.min.js` and searching on the Umbraco GitHub repo, found it to be the introduction of a `propertyEditorDisabled` function...

https://github.com/umbraco/Umbraco-CMS/blob/release-10.2.0/src/Umbraco.Web.UI.Client/src/common/directives/components/content/umbtabbedcontent.directive.js#L194

The `$scope.contentNodeModel` object assumes to have a `.variants` property, _which Content Blocks doesn't have_, so it throws an error.

### Steps to reproduce

On an Umbraco instance that uses the Content Blocks editor, upgrade to Umbraco v10.2.0.
Trying to view a Content Block item's properties in the overlay panel editor, none of the property editors will be displayed.

### Expected result / actual result

I'd expect the property editors to be displayed in the Content Blocks overlay panel editor.

### Do you have Umbraco ModelsBuilder enabled?

- [ ] Yes, it is enabled.

### What browsers are you seeing the problem on?

Chrome

**Comment by leekelleher** [comment]
I'll to get a hot-fix patch out soon (early next week).

In the meantime, if someone needs a patch sooner, then they can manually edit the "~/App_Plugins/Contentment/editors/content-blocks.overlay.html" file.

[Line 119](https://github.com/leekelleher/umbraco-contentment/blob/4.1.3/src/Umbraco.Community.Contentment/DataEditors/ContentBlocks/content-blocks.overlay.html#L119), from this...

```
<umb-tabbed-content content="vm.content"></umb-tabbed-content>
```

...to this...

```
<umb-tabbed-content content="vm.content" content-node-model="{ variants: [] }"></umb-tabbed-content>
```

**Comment by leekelleher** [comment]
Fixed in [Contentment v4.1.4](https://github.com/leekelleher/umbraco-contentment/releases/tag/4.1.4).
